src/Components/CategoryFilter.js

```diff
@@ -0,0 +1,7 @@
+/* eslint-disable react/prop-types */
```

Comment:
With the ESLint inline comment, you are indirectly changing the linter's rules by disabling these rules, let's fix the codebase instead.

Perhaps adding the prop-types would fix the error.